doc/source/design.rst

```diff
@@ -335,10 +335,21 @@ Hooks are passed through Aura as an ADT of functions.
 Aesthetics
 ----------
 
+Size Information
+~~~~~~~~~~~~~~~~
+If ``--verbose-size`` is passed to Aura, the following information is
+displayed before installation from the official repositories (may not be
+possible for AUR)
```

Comment:
The `AUR` line isn't needed here, as not all Aura backends with have access to the AUR. Only Arch ones will.